demo/react-hooks-controlled-multiple.html

```diff
@@ -68,45 +69,35 @@ <h2>Controlled stack</h2>
           });
         })
         .on('removed', (ev, gsItems) => {
+          /* Looks like a bug in GridStack */
```

Comment:
what do you mean ? 'removed' should only be called on item being removed | added, and change on existing ones changing values. all 3 cases need to update your list - which is why I don't like have 2 list to sync and is not the right way...

Reply:
There is one case when you move one widget from one grid into another and when the widget is removed form the original grid, a relayout  occurs but the change event is not triggered despite the fact that some widgets changed their position, so i have to manually check this and call change handler to have the relevant layout

Reply:
that sounds like a bug - please post one. I just tried [two.html](https://gridstackjs.com/demo/two.html) dragging from the right side (where float=false) and item that would cause an update and indeed grid1 doesn't trigger a change event like it should!